CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 [comment]: <> "NOTE! Ensure all images are added via the \[label\]\(link\) syntax!"
 
+## 2023-04-13
+| What's new | |
+| :--- | :-: |
+| **📓 Schema Checks in Launches have been removed.** Studio no longer supports Schema Checks as a part of Launches. We strongly recommend that users use rover and our full featured Checks offering to prevent Launches which will negatively impact your traffic, as this is the best way to control changes to your SuperGraph. If you would like to see Checks and change management in Launches, please get ahold of your account rep, we'd love to hear from you! | |
```

Comment:
nitpick: Other instances in this [changelog file](https://github.com/apollographql/apollo-studio-community/blob/cd4af7f44b64198249e5eb8c4e3450b73fa802e1/CHANGELOG.md?plain=1#L19-L39) and our [marketing](https://www.apollographql.com/supergraph/) don't have any capitalization around `SuperGraph`